### PR TITLE
Add feature: sector offset

### PR DESCRIPTION
**Pull request by lubyant**
Apply for the issue #235 
Add an argument for the method such that the sector can be edited flexibly.
For example

Picture1

By default (left), the first sector is [-11.25, 11.25].

```
dir = [360*random.random() for _ in range(400)]
spd = [5*random.random() for _ in range(len(dir))]
ax.bar(dir, spd, bins=bins, nsector=16)
```

Create an 11.25 offset for the sector (right), the first sector is [0, 22.5] and also for the other sectors. And, the bar is rotate 11.25 degrees.
```
dir = [360*random.random() for _ in range(400)]
spd = [5*random.random() for _ in range(len(dir))]
ax.bar(dir, spd, bins=bins, nsector=16, sectoroffset=11.25)
```

The same thing applies to the box plot,
Picture2

### CODE REVIEW

**Comment by ocefpaf** [comment]
The code is solid and does what you need.

I'm not an expert here so can you explain a bit more on the use cases for this change? If we cannot get more experienced devs here to review I'm tempted to merge b/c this doesn't disrupt the conventional use anyway.

**Comment by akrherz** [comment]
This change appears to be a convenient way to enforce the "engineering" convention for windroses whereby the bars represent the direction the wind is blowing toward and not from (meteorology).  Reviewing the PR, I do notice a spelling typo with `offsect`

**Comment by lubyant** [comment]
Hi @ocefpaf,
I have made a few changes to incorporate the change in the histogram. Here is what it looks like. Can you review my pull request?
image
image

File: windrose/windrose.py [hunk 1]
```diff
@@ -575,6 +578,12 @@ def bar(self, direction, var, **kwargs):
             number of sectors used to compute the windrose table. If not set,
             nsector=16, then each sector will be 360/16=22.5°, and the
             resulting computed table will be aligned with the cardinals points.
+        sectoroffset: float, optional
+            the offset for the sectors between [-180/nsector, 180/nsector].
+            By default, the offsect is zero, and the first sector is
```
Comment (ocefpaf) [inline windrose/windrose.py:583]:
```suggestion
            By default, the offset is zero, and the first sector is
```

File: windrose/windrose.py [hunk 2]
```diff
@@ -791,11 +820,16 @@ def histogram(direction, var, bins, nsector, normed=False, blowto=False, total=0
...
+        -angle / 2 + sectoroffset,
+        360.0 + angle + sectoroffset,
+        angle,
+        dtype=float,
+    )
     dir_edges = dir_bins.tolist()
     dir_edges.pop(-1)
     dir_edges[0] = dir_edges.pop(-1)
-    dir_bins[0] = 0.0
+    # dir_bins[0] = 0.0 + sectoroffset
```
Comment (ocefpaf) [inline windrose/windrose.py:832]: Why is this commented out? Do we need it for your feature?

**Review by ocefpaf — APPROVED** [review]
LGTM, I'll leave it here just a bit more to get a second opinion. If no one can look at it in the next week I'll merge it as-is.

Thanks for the PR!

**Pull request merged**